CONTRIBUTING.md

```diff
@@ -0,0 +1,171 @@
+# Contributing Guide
+
+* [New Contributor Guide](#contributing-guide)
+  * [Ways to Contribute](#ways-to-contribute)
+  * [Find an Issue](#find-an-issue)
+  * [Ask for Help](#ask-for-help)
+  * [Pull Request Lifecycle](#pull-request-lifecycle)
+  * [Development Environment Setup](#development-environment-setup)
+    * [Requirements](#requirements)
```

Comment:
These links under #development-environment-setup section seem not to lead anywhere ...